backends/ebpf/ebpftargets.py

```diff
@@ -168,58 +193,67 @@ def interface_of_filename(self, f):
                    lstrip(self.pcapPrefix).rsplit('_', 1)[0])
 
     def compile_p4(self, argv):
-        ''' To override '''
-        ''' Compile the p4 target '''
+        """ To override """
```

Comment:
I think that this comment should not be the function documentation, it should be a separate comment with `#`